calico/felix/actor.py

```diff
@@ -232,11 +249,28 @@ def _step(self):
                 except BaseException as e:
                     _log.exception("Exception processing %s", msg)
                     results.append(ResultOrExc(None, e))
+                    message_exception = e.__class__.__name__ +\
+                                                      ': ' + str(e)
                     _stats.increment("Messages executed with exception")
                 else:
+                    message_exception = None
                     results.append(ResultOrExc(result, None))
                     _stats.increment("Messages executed OK")
                 finally:
+                    batch_uuids.add(msg.uuid)
+                    # Log the message info
+                    # Strings are enclosed in quotes. Numbers aren't.
+                    _message_log.info('"{uuid}": {data}'.format(
```

Comment:
This will do all the work to format this message even if logging isn't enabled, including translating the object to JSON.  That'll add a big overhead.  Since you're dumping the JSON, which requires a function call, you could guard this call with `if _message_log.isEnabledFor(logging.INFO):` so that it doesn't do any work if logging is turned off.